jsonmerge_utils.py

```diff
@@ -5,17 +5,38 @@
 @author: GRENTOR
 """
 import os
+import argparse
 import logging
 #from tqdm import tqdm
 from merge_files import Merge
+
+parser = argparse.ArgumentParser(description="link-check is a broken link identifier")
+parser.add_argument('-ip',
+                    help="Input prefix",
+                    required=True)
+parser.add_argument('-op',
+                    help="Output prefix",
+                    required=True)
+parser.add_argument('-maxFileSize',
+                    help="The maximum file size (in bytes) that each merged file should have",
+                    required=True)
+parser.add_argument('-filepath',

```

Comment:
`dest = "data_dir"`

also `-filepath` doesnt seem intuitive since we are actually asking for directory of where all files to be processed are stored, filepath might give user the wrong idea.
could you think of changing it to something else ?

Reply:
Like you said, -filepath does give off the wrong message. I've changed it to dir_path, which makes a lot more sense.